prow/plugins.yaml

```diff
@@ -73,6 +69,7 @@ approve:
   - kubernetes-incubator/ip-masq-agent
   require_self_approval: false
   lgtm_acts_as_approve: true
+  ignore_review_state: true
```

Comment:
Please comment that this is not an explicit choice, just inherited default behavior